tf-modules/cumulus/outputs.tf

```diff
@@ -1,9 +1,16 @@
 # API outputs
 
-output "distribution_bucket_map" {
-  value = module.distribution.distribution_bucket_map
-}
+# TEA-Specific outputs. Uncomment to use TEA distribution
```

Comment:
See above. We probably need these values active.

Reply:
Yes they'll need to be conditional

Reply:
And they are: https://github.com/nasa/cumulus/pull/2300/files#diff-c6fa420df06c6c4749e5fcb023d8b4a4096a0050767a3814d021d039571717ebR3